kernel/content/edit.php

```diff
@@ -775,7 +775,7 @@ function computeRedirect( $module, $object, $version, $EditLanguage = false )
                                 }
                                 if ( isset( $result['path'] ) )
                                 {
-                                    $Result['path'] = $result['path'];
+                                    $Result = array('path' => $result['path']);
```

Comment:
It's potentially similar issue below.

Safer and more readable would be to set $Result = array(); at the start of `else if ( isset( $operationResult['result'] ) )` block.

Reply:
Thanks for the feedback. I moved $Result to the top of the case.